core/webawallet/config/config.js

```diff
@@ -0,0 +1,40 @@
+import WebaverseABI from "../abi/Webaverse.json";
+import WebaverseERC1155ABI from "../abi/WebaverseERC1155.json";
+import WebaverseERC20ABI from "../abi/WebaverseERC20.json";
+import WebaverseAccountsABI from "../abi/Accounts.json";
+
+export const config = {
+  sidechainURL: "https://13.57.177.184:8545",
```

Comment:
Do not hardcode any addresses.

Problematic code comes from upstream.

Reply:
and where should it be taken from ?

Reply:
DNS.